front-end-node/main/MainOverrides.js

```diff
@@ -38,6 +39,18 @@ WebInspector.MainOverrides.prototype = {
     };
   },
 
+  _exposeSourceMaps: function() {
```

Comment:
Instead of this we can implement:
```js
_getTargetData: function() {
  var entries = WebInspector.debuggerWorkspaceBinding._targetToData.entries(),
    entry,
    key,
    target;

  while((entry= entries.next()) && entry.value) {
    key = entry.value[0];
    target = entry.value[1];
    if (key .name() == 'Main') break;
  },
  
  return target;
}
```